### PR TITLE
Ability to skip when both decks are playing on auto dj

### PR DESCRIPTION
**Pull request by crisclacerda**
This PR fixes this https://bugs.launchpad.net/mixxx/+bug/1399974?comments=all

Now when 2 decks are playing it assumes the user wants to remove the next track on the auto dj playlist.

### CODE REVIEW

**Comment by daschuer** [comment]
Thank you for your first PR. 

Unfortunately a commit hook on Travis fails: 
```
Trim Trailing Whitespace.................................................Failed
```

You may also install the commit hooks locally if you like: 
https://github.com/mixxxdj/mixxx/blob/master/.pre-commit-config.yaml
Unfortunately this was only tested on Linux, so you may discover issue on MacOs. 

Unfortunately another test fails as well: 
```
206 - EngineMicrophoneTest.TestInputMatchesOutput (Child aborted) 
```
Likely a false positive, so we can ignore it here. I sorry about that. Our CI is not that reliably. We need to work on it but that is another topic.

**Comment by daschuer** [comment]
Yes probably. 
@

**Comment by daschuer** [comment]
Sorry the close and comment button strikes me again.
@crisclacerda please rebase your branch to 2.2
And forth push it. This will update this PR.
Than we can change the target branch to 2.2

**Comment by Holzhaus** [comment]
In case you don't know how to rebase: https://mixxx.org/wiki/doku.php/using_git#targeting_another_base_branch

**Comment by uklotzde** [comment]
`git rebase --onto 2.2 49372742c5637a71b833d84695c95aaa8b775608^`

**Comment by crisclacerda** [comment]
Made changes, commented on launchpad post and signed document.

**Comment by crisclacerda** [comment]
Hm, it failed again to build on CI. Now it looks like it's trying to use python 3 when building on mac OS.
Sorry about that, I am not familiar with these systems.

```
File "/Users/travis/build/mixxxdj/mixxx/build/osx/OSConsX.py", line 36
    print s,
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(s, end=" ")?
```

File: src/library/autodj/autodjprocessor.cpp [hunk 1]
```diff
@@ -234,6 +234,18 @@ AutoDJProcessor::AutoDJError AutoDJProcessor::skipNext() {
     } else if (!rightDeck.isPlaying()) {
         removeLoadedTrackFromTopOfQueue(rightDeck);
         loadNextTrackFromQueue(rightDeck);
+    } else {
```
Comment (daschuer) [inline src/library/autodj/autodjprocessor.cpp]: Here is the tailing white space error, Travis complains about.

**Comment by daschuer** [comment]
I think in this case we can ignore most of the CI build issues. This is all a bit legacy. 
I am just building locally and if it works successful we can merge.

**Comment by uklotzde** [comment]
The 2.2 CI builds are broken depending on the platform and won't be fixed. These are known issues. We need to abandon the 2.2 branch with a final release asap.

**Comment by uklotzde** [comment]
Please add a descriptive line with a reference to the bug to the CHANGELOG

**Comment by daschuer** [comment]
LGTM Thank you.

**Comment by crisclacerda** [comment]
Nice, thanks for guiding me through my first PR and sorry for any inconvenience, hopefully, next ones will be more smooth =)

**Comment by daschuer** [comment]
To be honest, this was quite smooth, every complains where of type tap doors.
I am sorry for that. Carry on. :-)

**Pull request merged**